static/app/utils/number/rangeMap.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import {RangeMap} from './rangeMap';
+
+describe('RangeMap', () => {
+  test('matches an integer range', () => {
+    const ladder = new RangeMap([
+      {min: 0, max: 10, value: 'first'},
+      {min: 10, max: 20, value: 'second'},
+      {min: 20, max: 50, value: 'third'},
```

Comment:
This leaves room for inclusivity question. How about something like this to indicate that values are inclusive?
```suggestion
      {min: 0, max: 10, value: 'first'},
      {min: 11, max: 20, value: 'second'},
      {min: 21, max: 50, value: 'third'},
```

Reply:
I'm a little wary of that because it only works for integers, and the class doesn't do anything to enforce integer-only use. Should it? It seems like there's a use-case for matching ranges to floats though I don't see it in the code right now.

Another option is maybe to make the boundary matching explicit, i.e., allow configuring which boundary is inclusive, and have it be `min` by default